cmd/template/cluster/provider/capg.go

```diff
@@ -182,6 +182,11 @@ func templateDefaultAppsGCP(ctx context.Context, k8sClient k8sclient.Interface,
 			Namespace:               organizationNamespace(config.Organization),
 			Version:                 appVersion,
 			UserConfigConfigMapName: configMapName,
+			DefaultingEnabled:       false,
```

Comment:
I'm not sure what this implies, but without setting this to `false` the `config` part of the CR is removed.